scripts/start-workers.sh

```
@@ -0,0 +1,319 @@
+
+    # Start ML Celery Worker
+    echo -e "${YELLOW}Starting ML Celery Worker (hubspot-webhook-events, fatigue-monitoring, budget-optimization)...${NC}"
+    celery -A src.celery_app worker \
+        --loglevel=info \
+        --concurrency=4 \
+        --queues=hubspot-webhook-events,fatigue-monitoring,budget-optimization \
+        --logfile="${LOG_DIR}/ml-worker.log" \
+        --pidfile="${LOG_DIR}/ml-worker.pid" \
+        --detach
+
+    if [ $? -eq 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)